### ISSUE TITLE
Remove History.lat and History.lon

### ISSUE THREAD

**Issue by jameshiebert** [pacificclimate/pycds#199]
A well normalized set of tables should have one single source of information. Unfortunately, the `meta_history` table has *both* numeric lat/lon attributes in addition to a PostGIS `geometry` object. This has sown confusion, in that data managers tend to forget to update the `geometry` object:

```sql
crmp=> select count(*) from meta_history where the_geom is NULL and (lat is not null and lon is not null);
 count 
-------
   183
(1 row)
```

And [even our own developer have used lat/lon instead of the `geometry`](https://github.com/pacificclimate/station-data-portal-backend/blob/master/sdpb/api/histories.py#L84).

`lat` and `lon` should be removed.

**Comment by rod-glover** [comment]
Yes, that would be me for sure -- in many places, including the frontends, which will also need to be updated. It was not clear to me what the use (or reliability) of `geometry` was, and all existing code I encountered did not use it. 

Therefore I add the following much broader suggestion: Document these columns better. Their names alone are frequently not sufficient. Let's look into whether that documentation can be part of the SQLAlchemy column definition so that it makes it into the database proper as well.
